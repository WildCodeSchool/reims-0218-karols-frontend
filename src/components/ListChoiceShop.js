import React from "react"
import ChoiceShop from "./ChoiceShop"
import { Row, Col } from "reactstrap"

export default class ListChoiceShop extends React.Component {
  render() {
    return (
      <Row>
        <Col lg="4" className="text-center mb-3">
          <ChoiceShop
            city="Paris"
            adress="75000"
            image="https://images.unsplash.com/33/YOfYx7zhTvYBGYs6g83s_IMG_8643.jpg?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=82fecbde67b1966b4ea8118f3bea896e&auto=format&fit=crop&w=967&q=80"
          />
        </Col>
        <Col lg="4" className="text-center">
          <ChoiceShop
            city="Lyon"
            adress="69000"
            image="https://images.unsplash.com/photo-1496277195689-21125faa0eeb?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=789210ffc73d1822ce69059fcefcabf0&auto=format&fit=crop&w=1050&q=80"
          />
        </Col>
        <Col lg="4" className="text-center">
          <ChoiceShop
            city="Cannes"
            adress="06150"
            image="https://thumbs.dreamstime.com/z/ville-de-cannes-france-2627857.jpg"
          />
        </Col>
      </Row>
    )
  }
}
