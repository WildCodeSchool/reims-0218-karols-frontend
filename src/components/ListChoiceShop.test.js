import React from "react"
import ReactDOM from "react-dom"
import ListChoiceShop from "./ListChoiceShop"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ListChoiceShop />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ListChoiceShop />).toJSON()
  expect(tree).toMatchSnapshot()
})
