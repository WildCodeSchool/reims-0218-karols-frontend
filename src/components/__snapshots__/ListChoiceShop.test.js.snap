// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="text-center mb-3 col-lg-4"
  >
    <div
      className="mb-4"
    >
      <div
        className="card"
      >
        <div
          className="wrapper text-center"
          style={
            Object {
              "backgroundImage": "url(https://images.unsplash.com/33/YOfYx7zhTvYBGYs6g83s_IMG_8643.jpg?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=82fecbde67b1966b4ea8118f3bea896e&auto=format&fit=crop&w=967&q=80)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            }
          }
        >
          <div
            className="data"
          >
            <div
              className="content card-body"
            >
              <h5
                className="city card-title"
                style={
                  Object {
                    "fontWeight": "500",
                    "marginTop": "10px",
                  }
                }
              >
                <p
                  style={
                    Object {
                      "color": "#FFF",
                    }
                  }
                >
                  Paris
                </p>
              </h5>
              <button
                className="btn btn-secondary"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#fff",
                    "display": "block",
                    "fontSize": "12px",
                    "fontWeight": "700",
                    "lineHeight": "1",
                    "margin": "2em auto 1em",
                    "marginBottom": "40px",
                    "padding": "10px 20px",
                    "position": "relative",
                    "textAlign": "center",
                  }
                }
                type={undefined}
              >
                Sélectionner
              </button>
            </div>
            <div
              className="card-footer"
              style={
                Object {
                  "backgroundColor": "rgba(136,136,136, 0.5)",
                }
              }
            >
              <p
                className="text"
                style={
                  Object {
                    "height": "70px",
                    "margin": "0",
                  }
                }
              >
                75000
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="text-center col-lg-4"
  >
    <div
      className="mb-4"
    >
      <div
        className="card"
      >
        <div
          className="wrapper text-center"
          style={
            Object {
              "backgroundImage": "url(https://images.unsplash.com/photo-1496277195689-21125faa0eeb?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=789210ffc73d1822ce69059fcefcabf0&auto=format&fit=crop&w=1050&q=80)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            }
          }
        >
          <div
            className="data"
          >
            <div
              className="content card-body"
            >
              <h5
                className="city card-title"
                style={
                  Object {
                    "fontWeight": "500",
                    "marginTop": "10px",
                  }
                }
              >
                <p
                  style={
                    Object {
                      "color": "#FFF",
                    }
                  }
                >
                  Lyon
                </p>
              </h5>
              <button
                className="btn btn-secondary"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#fff",
                    "display": "block",
                    "fontSize": "12px",
                    "fontWeight": "700",
                    "lineHeight": "1",
                    "margin": "2em auto 1em",
                    "marginBottom": "40px",
                    "padding": "10px 20px",
                    "position": "relative",
                    "textAlign": "center",
                  }
                }
                type={undefined}
              >
                Sélectionner
              </button>
            </div>
            <div
              className="card-footer"
              style={
                Object {
                  "backgroundColor": "rgba(136,136,136, 0.5)",
                }
              }
            >
              <p
                className="text"
                style={
                  Object {
                    "height": "70px",
                    "margin": "0",
                  }
                }
              >
                69000
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="text-center col-lg-4"
  >
    <div
      className="mb-4"
    >
      <div
        className="card"
      >
        <div
          className="wrapper text-center"
          style={
            Object {
              "backgroundImage": "url(https://thumbs.dreamstime.com/z/ville-de-cannes-france-2627857.jpg)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            }
          }
        >
          <div
            className="data"
          >
            <div
              className="content card-body"
            >
              <h5
                className="city card-title"
                style={
                  Object {
                    "fontWeight": "500",
                    "marginTop": "10px",
                  }
                }
              >
                <p
                  style={
                    Object {
                      "color": "#FFF",
                    }
                  }
                >
                  Cannes
                </p>
              </h5>
              <button
                className="btn btn-secondary"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#fff",
                    "display": "block",
                    "fontSize": "12px",
                    "fontWeight": "700",
                    "lineHeight": "1",
                    "margin": "2em auto 1em",
                    "marginBottom": "40px",
                    "padding": "10px 20px",
                    "position": "relative",
                    "textAlign": "center",
                  }
                }
                type={undefined}
              >
                Sélectionner
              </button>
            </div>
            <div
              className="card-footer"
              style={
                Object {
                  "backgroundColor": "rgba(136,136,136, 0.5)",
                }
              }
            >
              <p
                className="text"
                style={
                  Object {
                    "height": "70px",
                    "margin": "0",
                  }
                }
              >
                06150
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
